src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -114,12 +114,12 @@ private File createTempFile(String prefix, String suffix) throws IOException {
 
     private static void invokeFireAuditFinished(Checker checker)
             throws ReflectiveOperationException {
-        TestUtil.invokeMethod(checker, "fireAuditFinished");
+        TestUtil.invokeMethodDefault(checker, "fireAuditFinished");
```

Comment:
`Default` is not clear.

please name method `invokeVoidMethod` same for other method
<img width="865" height="623" alt="Image" src="https://github.com/user-attachments/assets/98396aeb-3feb-4fd5-a153-147e9e662134" />